entity/entity.h

```diff
@@ -18,6 +18,17 @@ class RenderPass;
 
 class Entity {
  public:
+  /// All pipeline blend mode presets assume that both the source (fragment
+  /// output) and destination (first color attachment) have colors with
+  /// premultiplied alpha.
+  enum class PipelineBlendMode {
```

Comment:
Nit: Just `BlendMode` should be fine. That it is affected using a pipeline variant is of no concern to the user of the entity framework.

Reply:
Done.